src/common/get-configuration-file-model/get-configuration-file-model.global-override-for-injectable.ts

```diff
@@ -21,17 +20,20 @@ export default getGlobalOverride(getConfigurationFileModelInjectable, (di) => {
     assert(options.configName, "Missing options.configName");
 
     const configFilePath = path.posix.join(options.cwd, options.configName);
+    let store: object = {};
+
+    try {
+      store = readJsonSync(configFilePath);
+    } catch {
+      // ignore
+    }
```

Comment:
These make me uneasy. You? :)

Reply:
_Offtopic_: such a long filename with many meanings kinda hard to perceive `get-configuration-file-model.global-override-for-injectable.ts`. Could we have more clear/simple `(file|class?)` names?

Reply:
At least global-override-for-yaddayaddayadda will get shorter soon enough. Outside of that, naming should follow common sense, or what is the proposal? :)